src/rpc.rs

```diff
@@ -1470,11 +1491,11 @@ mod tests {
                     let bobs_request: RequestGuard = bob
                         .client
                         .create_request(channel)
-                        .with_payload(large_payload.clone())
```

Comment:
If I understand a1c0b0fc7a871500bb2cc6431f0ab04c952b7aaa correctly, the issue is that if Alice sends fast enough, Bob needs to call `next_event` to clear Bob's `queue_for_sending`, but can't?

Reply:
Exactly.  Took me quite a while to get there!